src/fides/core/dataset.py

```diff
@@ -400,6 +401,10 @@ def get_snowflake_schemas(
     and being able to fall back to using the connector.
 
     Reference: https://github.com/snowflakedb/snowflake-sqlalchemy/issues/157
+
+    Due to performance issues resulting in FastAPI timeouts, parallelization is
+    no used. The default number of threads to be used is 4 to cover most existing
```

Comment:
nit: is this supposed to be `not` or `now`? i think it's now - since it looks like we've switched this to use parallelization1 

```suggestion
    now used. The default number of threads to be used is 4 to cover most existing
```

Reply:
definitely now! thanks 👍🏽